### ISSUE TITLE
Add fonts for Church Slavonic support

### ISSUE THREAD

**Issue by typiconman** [google/fonts#2403]
Add Ponomar Unicode font, which would provide the necessary glyphs for Church Slavonic.

* The repository is here: https://github.com/typiconman/fonts-cu
* The binary fonts are here: https://sci.ponomar.net/fonts.html

It would be nice to add some of the other fonts as well, but at least Ponomar Unicode is needed for proper support of Church Slavonic.

**Comment by typiconman** [comment]
Any chance this can get resolved some time soon?

**Comment by typiconman** [comment]
Biweekly ping.

**Comment by typiconman** [comment]
Still nothing on this issue. Too bad; the lack of an OCS font makes it very difficult to use Google Docs to collaborate on OCS projects.

**Comment by typiconman** [comment]
Quarterly ping.

**Comment by davelab6** [comment]
@typiconman emailed fonts@ and mentioned https://github.com/slavonic/Pochaevsk is now following the template. @chrissimpkins could we queue this up for Q4?

**Comment by typiconman** [comment]
I created also a repository for the [Triodion font](https://github.com/slavonic/Triodion).

**Comment by davelab6** [comment]
I see #7982 now - thanks!

**Comment by typiconman** [comment]
Also added Shafarik, see #8012.

**Comment by typiconman** [comment]
Added Ponomar, see #8156.

**Comment by yanone** [comment]
Asked author by email about how many more of the fonts we want to onboard (to create new sub-issues, or otherwise close the issue out.)

**Comment by yanone** [comment]
Author replied that they want to land one or two more Slavonic fonts under the umbrella of this priority issue, then we'll wrap it up.

**Comment by typiconman** [comment]
Added Monomakh, please see #8982
@yanone Please review.

**Comment by yanone** [comment]
Update: Other than Monomakh (see comment above), the author notified me by email that they want to add one more font called Mezenets for Church Salvonic musical notation, and with that we’ll wrap the Slavonic session up.

**Comment by davimmt** [comment]
Looking forward to it. Specially with diacritical marks.

![Image](https://github.com/user-attachments/assets/5d00648f-637d-47e2-8056-b46197333c61)

**Comment by chrissimpkins** [comment]
Is this parent tracking issue across the multiple Church Slavonic families now complete? OK to close?

**Comment by yanone** [comment]
@chrissimpkins No, I’m waiting for one more Church Slavonic font for musical notations. A sub-issue doesn't yet exist for it. I'll ask the author where they stand on this.

**Comment by yanone** [comment]
@typiconman Where do we stand on the last font you said you want to onboard?

**Comment by typiconman** [comment]
Yes, I got distracted by a different font project, but I will create a repository for you to onboard asap.

**Comment by chrissimpkins** [comment]
Let's close this parent issue as completed for Q1 and open a new single project issue for the final family.  Thanks!
